packages/grafana-ui/src/types/graph.ts

```diff
@@ -8,4 +9,7 @@ export interface GraphSeriesXY {
   color: string;
   data: GraphSeriesValue[][]; // [x,y][]
   info?: DisplayValue[]; // Legend info
+  stats: ColumnStats;
+  isVisible: boolean;
+  useRightYAxis: boolean;
```

Comment:
was wondering if yAxis: number would be more future proof (if we ever wanted more than 2).

Reply:
something with the naming that makes it feel like an action :)

Reply:
@torkelo how could you have more than 2 y axis on a single graph?

Reply:
Regarding `useRightYAxis` I agree, the name ain't the best one. `displayOnRightYAxis`?

Reply:
@dprokop -- see: https://plot.ly/javascript/multiple-axes/
![image](https://user-images.githubusercontent.com/705951/55969937-7e167800-5c33-11e9-9ac4-744d6bb6b516.png)

Sometimes done on the x axis, but rare and pretty weird:
![image](https://user-images.githubusercontent.com/705951/55970071-be75f600-5c33-11e9-9170-c6ad7b4c0c28.png)

Reply:
Thanks @ryantxu!